LyricFy/Views/Components/Home/ProjectsCell.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  ProjectsCell.swift
+//  LyricFy
+//
+//  Created by Marcos Costa on 28/04/23.
+//
+
+import UIKit
+
+class ProjectsCell: UICollectionViewCell {
+    
+    static let identifier: String = "ProjectsCell"
+        
+    lazy var projectComponent: UIView = {
+        let projectComponent = UIView()
+        projectComponent.translatesAutoresizingMaskIntoConstraints = false
+        projectComponent.backgroundColor = .lightGray
+        projectComponent.layer.cornerRadius = 10
+        return projectComponent
+    }()
+    
+    lazy var nameProject: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 20)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    lazy var date: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 11)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    lazy var stackView: UIStackView = {
+        let stack = UIStackView()
+        stack.axis = .vertical
+        stack.spacing = 5
+        stack.alignment = .center
+        stack.distribution = .fillEqually
+        stack.translatesAutoresizingMaskIntoConstraints = false
+        [nameProject, date].forEach { stack.addArrangedSubview($0) }
+        return stack
+    }()
+    
+    override init(frame: CGRect) {
```

Comment:
Não precisa sobrepor o init com o parâmetro frame se o frame for atribuído como .zero em toda inicialização